### PR TITLE
add: `wezterm-bin`

### PR DESCRIPTION
**Pull request by smokeythemonkey**
## Progress 

- [x] Edit packagelist
- [x] Add initial pacscript
- [ ] Contact devs
- [ ] Add maintainer to pacscript

### CODE REVIEW

File: packages/wezterm-bin/wezterm-bin.pacscript [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+version="20230326-111934-3666303c"
+description="A GPU-accelerated cross-platform terminal emulator and multiplexer written by @wez and implemented in Rust"
+url="https://github.com/wez/wezterm/releases/download/${version}/wezterm-${version}.Ubuntu22.04.tar.xz"
+hash="636428e0faabd230f3be931131e47c708b1e0451ccce7a80e281346923c9b089"
+breaks="${pkgname}-app"
+repology=("project: wezterm")
+maintainer="smokeythemonkey <smokeythemonkey@posteo.net>"
+
+install() {
+
```
Comment (Henryws) [inline packages/wezterm-bin/wezterm-bin.pacscript:13]:
```suggestion
```

**Review by Henryws — CHANGES_REQUESTED** [review]

**Comment by Henryws** [comment]
You didn't run `pre-commit` on the repo.

File: packages/wezterm-bin/wezterm-bin.pacscript [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  sudo rm -f "/usr/share/bash-completions/wezterm"
+  sudo rm -f "/usr/share/icons/hicolor/128x128/apps/org.wezfurlong.wezterm.png"
+  sudo rm -f "/usr/share/metainfo/org.wezfurlong.wezterm.appdata.xml"
+  sudo rm -f "/usr/share/nautilus-python/extensions/wezterm-nautilus.py"
+  sudo rm -rf "/usr/share/wezterm/"
+  sudo rm -f "/usr/share/zsh/functions/Completion/Unix/_wezterm"
+
+  # Delete wezterm config
+  if ask "Do you want to delete wezterm config files?" N; then
+    rm -rf ~/.config/.wezterm/
```
Comment (Henryws) [inline packages/wezterm-bin/wezterm-bin.pacscript:43]:
```suggestion
    rm -rf "${homedir}/.config/.wezterm/"
```

**Review by Henryws — CHANGES_REQUESTED** [review]
Use the Pacstall variable `homedir` to get the home directory instead.

**Comment by smokeythemonkey** [comment]
Will change this in wezterm-app when theres another update

File: packages/wezterm-bin/wezterm-bin.pacscript [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+  # Copy Icons and plugins
+  sudo cp -r "${SRCDIR}/wezterm/usr/share/"* "/usr/share/"
+
+  # Install .desktop file
+  sudo install -Dm644 "${SRCDIR}/wezterm/usr/share/applications/org.wezfurlong.wezterm.desktop" "${pkgdir}/usr/share/applications/org.wezfurlong.wezterm.desktop"
+}
+
+postinst() {
+  sudo chmod +x "${SRCDIR}/wezterm/etc/profile.d/wezterm.sh"
```
Comment (Henryws) [inline packages/wezterm-bin/wezterm-bin.pacscript:28]: Why are you using `SRCDIR` outside of `{prepare,build,install}`?

**Review by Henryws — CHANGES_REQUESTED** [review]

File: packages/wezterm-bin/wezterm-bin.pacscript (hunk 3 above)
Comment (smokeythemonkey) [reply packages/wezterm-bin/wezterm-bin.pacscript:28]: I tried pkgdir stowdir and stgdir, none of these worked. When looking for where the zip is downloaded and extracted to, SRCDIR was the only location where the files were available.

Comment (smokeythemonkey) [reply packages/wezterm-bin/wezterm-bin.pacscript:28]: i guess i could install the file first then run it....

**Review by Henryws — APPROVED** [review]

**Pull request merged**